test/routes/issueRoutesTest.js

```diff
@@ -159,12 +149,10 @@ describe('Issue requests', function(){
     describe('Delete issue', function(){        
 
         describe('Valid params', function() {
-
-            let issue;
-
+               
             it('should delete the issue', function(done) {
                 request('http://localhost:8000')
-                .delete('/issues/5aba5d0301f3703b5c720459')
+                .delete('/issues/' + testIssueId)
```

Comment:
Nice... Try formatting this like this with backticks

\`/issues/${testIssueId}\`